microsoft-edge/webview2/concepts/developer-guide.md

```diff
@@ -10,59 +10,65 @@ ms.date: 08/03/2021
 ---
 # Development best practices for WebView2 apps
 
-Every development team follows different practices when building their application.  When you build WebView2 production apps, we recommend following these recommendations and best practices.
+We recommend the following best practices for developing production WebView2 apps:
 
+## Use the Evergreen WebView2 Runtime
 
-<!-- ====================================================================== -->
-## Use the Evergreen Runtime (recommended)
+**We recommend using the Evergreen WebView2 Runtime for most WebView2 apps.**  The Evergreen runtime updates automatically on the client, so that the latest features and security patches are available to your WebView2 app.  The Evergreen runtime also requires less storage space on the disk than the Fixed Version runtime. Fixed Version runtime distribution is only recommended for apps that have strict compatibility requirements. For more details on the benefits, see [Details about the Evergreen Runtime distribution mode](../concepts/distribution.md#details-about-the-evergreen-runtime-distribution-mode).
+ 
+### Evergreen distribution checklist
 
-We generally recommend using the Evergreen WebView2 Runtime.  Fixed Version runtime distribution is only recommended for apps that have strict compatibility requirements.  The Evergreen runtime updates automatically on the client, so that the latest features and security patches are available to your WebView2 app.  The Evergreen runtime also requires less storage space on the disk than the Fixed Version runtime.
+#### 1. Ensure that the Evergreen WebView2 Runtime is installed on the client before your app uses the WebView2 control.  
+For more information, see [Detect if a suitable WebView2 Runtime is already installed](../concepts/distribution.md#detect-if-a-suitable-webview2-runtime-is-already-installed).
 
-If you use the Evergreen runtime, before running your WebView2 app, test whether the Evergreen WebView2 Runtime is installed on the client.  See [Deploying the Evergreen WebView2 Runtime](../concepts/distribution.md#deploying-the-evergreen-webview2-runtime).
+#### 2. Manage new versions of the Evergreen Runtime
 
-<!-- ====================================================================== -->
-## Run compatibility tests regularly when using the Evergreen Runtime
+<!--TODO Consider if we want to move this API / description into distribution.md, and have an overview here instead. -->
```

Comment:
move this API / description into distribution.md, and have an overview here instead?